tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -2256,56 +2106,34 @@ def test_crash_free_user_rate_orderby_crash_free_rate(self):
 
     def test_healthy_sessions(self):
         user_ts = time.time()
-        org_id = self.organization.id
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": (user_ts // 60) * 60,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "errored_preaggr"),
-                        self.release_tag: rh_indexer_record(org_id, "foo"),
-                    },
-                    "type": "c",
-                    "value": 4,
-                    "retention_days": 90,
-                },
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": user_ts,
-                    "tags": {
-                        self.session_status_tag: rh_indexer_record(org_id, "init"),
-                        self.release_tag: rh_indexer_record(org_id, "foo"),
-                    },
-                    "type": "c",
-                    "value": 10,
-                    "retention_days": 90,
-                },
-            ],
-            entity="metrics_counters",
-        )
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_error_metric,
-                    "timestamp": user_ts,
-                    "tags": {tag: value},
-                    "type": "s",
-                    "value": numbers,
-                    "retention_days": 90,
-                }
-                for tag, value, numbers in (
-                    (self.release_tag, rh_indexer_record(org_id, "foo"), list(range(3))),
-                )
-            ],
-            entity="metrics_sets",
-        )
+
+        for tag_value, release_tag_value, value, timestamp in (
```

Comment:
Here as well, maybe we can include the tags here so we need to only write one loop